mobile/android/base/android_strings.dtd

```diff
@@ -16,7 +16,7 @@
 <!ENTITY firstrun_bookmarks_subtext "Wyświetlaj wyniki z zakładek i historii podczas wyszukiwania.">
 <!ENTITY firstrun_data_title "Dane">
 <!ENTITY firstrun_data_message "Mniej danych, więcej oszczędności">
-<!ENTITY firstrun_data_subtext2 "Wyłącz obrazy, aby zmniejszyć zużycie danych.">
+<!ENTITY firstrun_data_subtext2 "Wyłącz obrazki, aby zmniejszyć zużycie danych.">
```

Comment:
It is a response to 'Personally, I would prefer "obraz" instead of "obrazek",
but the latter is what we have in every other place in Mozilla' [commit message](https://github.com/aviarypl/mozilla-l10n-central-pl/commit/9881f2aba0ea608beb085740921c85629d143c66).